### PR TITLE
chore(samples): fix BQ Dataset creation.

### PR DESCRIPTION
**Pull request by kchertenko**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-retail/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

**Review by parthea — CHANGES_REQUESTED** [review]
Please could you check the samples failures?

```
        if not 200 <= response.status_code < 300:
>           raise exceptions.from_http_response(response)
E           google.api_core.exceptions.Conflict: 409 POST https://bigquery.googleapis.com/bigquery/v2/projects/python-docs-samples-tests/datasets?prettyPrint=false: Already Exists: Dataset python-docs-samples-tests:my_random_dataset_name
```

See build log [here](https://source.cloud.google.com/results/invocations/36a5468e-da25-4af5-8541-c31638adc1b9/log).

File: samples/interactive-tutorials/events/setup_events/setup_cleanup_test.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# Copyright 2022 Google Inc. All Rights Reserved.
```
Comment (chenlei1216) [inline samples/interactive-tutorials/events/setup_events/setup_cleanup_test.py:1]: Let's remove the tests for the setup scripts as they were used one-off and may break the test data.

Comment (kchertenko) [reply samples/interactive-tutorials/events/setup_events/setup_cleanup_test.py]: ok. done

**Review by chenlei1216 — APPROVED** [review]

**Comment by kchertenko** [comment]
> Please could you check the samples failures?
> 
> ```
>         if not 200 <= response.status_code < 300:
> >           raise exceptions.from_http_response(response)
> E           google.api_core.exceptions.Conflict: 409 POST https://bigquery.googleapis.com/bigquery/v2/projects/python-docs-samples-tests/datasets?prettyPrint=false: Already Exists: Dataset python-docs-samples-tests:my_random_dataset_name
> ```
> 
> See build log [here](https://source.cloud.google.com/results/invocations/36a5468e-da25-4af5-8541-c31638adc1b9/log).

I removed the tests

**Review by parthea — APPROVED** [review]

**Pull request merged**